src/content/en/tools/workbox/modules/workbox-webpack-plugin.md

```diff
@@ -0,0 +1,82 @@
+project_path: /web/tools/workbox/_project.yaml
+book_path: /web/tools/workbox/_book.yaml
+description: The module guide for workbox-webpack-plugin.
+
+{# wf_updated_on: 2017-12-15 #}
+{# wf_published_on: 2017-12-15 #}
+{# wf_blink_components: Blink>ServiceWorker #}
+
+# Workbox webpack Plugin  {: .page-title }
+
+## What's the Workbox webpack Plugin?
+
+This is a plugin for the [webpack](https://webpack.js.org/) which generates a list of URLs to
+[precache](workbox-precaching) (known as a "precache manifest") based on the assets in a webpack
+compilation.
+
+## Considerations
+
+There are a few decisions to make which will determine the right configuration for your use case.
+
+### Should the plugin create your service worker file?
```

Comment:
This documentation describes what's currently implemented as of v3-alpha.3, but it feels clunky. See https://github.com/GoogleChrome/workbox/issues/1136 to track decisions about whether we should change this interface.